### ISSUE TITLE
Clicking on the card should have same effect as clicking the + sign

### ISSUE THREAD

**Issue by jmlord** [PEM-Humboldt/projects-se-ui#93]
In the booth, I saw many people clicking on the project card, and it does nothing. I do it too sometimes.

At least the title shoud have the same effect as the + button, then when unfolded the "go to page" button. Maybe even the whole card could do that.

**Comment by Angrypixels** [comment]
I agree to open the card in any point of the card. But the whole card should not open the project page since brings problems with copy text, that people sometimes would want to do, so would leave it in the bottom and the title
